lockfile/src/parsers/gem.rs

```diff
@@ -1,24 +1,196 @@
-use super::*;
+use nom::branch::alt;
+use nom::bytes::complete::{tag, take_until};
+use nom::character::complete::{line_ending, none_of, space0};
+use nom::combinator::recognize;
+use nom::error::{VerboseError, VerboseErrorKind};
+use nom::multi::{many1, many_till};
+use nom::sequence::{delimited, tuple};
+use nom::Err as NomErr;
```

Comment:
Is this rename actually necessary? Or do you just not like the look of `Err(Err::Failure(error))`?